client/plugins/fcc-source-challenges/create-challenge-nodes.js

```diff
@@ -1,6 +1,20 @@
 const crypto = require('crypto');
 const isEmpty = require('lodash/isEmpty');
 
+const FILE_CONTENTS = {
+  key: '',
+  ext: '',
+  name: '',
+  contents: '',
+  head: '',
+  tail: ''
+};
+
+const EMPTY_FILES = ['indexhtml', 'indexjs', 'indexjsx'].reduce((acc, curr) => {
+  acc[curr] = { ...FILE_CONTENTS };
+  return acc;
+}, {});
+
```

Comment:
```suggestion
const EMPTY_FILES = ['indexhtml', 'indexjs', 'indexjsx'].reduce(
  (acc, curr) => ({ ...acc, [curr]: FILE_CONTENTS })
, {});
```

Reply:
I like it.  Could you put this comment on the latest commit?  Github's telling me I can't commit the suggestion because it's 'outdated'.

Reply:
When I cleaned things up, I included this with one last tweak: it now clones `FILE_CONTENTS`.